### ISSUE TITLE
Remove "Knowledge Center" banner from support site

### ISSUE THREAD

**Issue by ReneeRendon** [rackerlabs/rackspace-how-to#1425]
Need to remove the "brain" banner from the support site.

Banner says: We've improved our Knowledge Center!
Our comprehensive Knowledge Center has been redesigned. We're calling it "How-To." Take a look!

**Comment by DonNateR** [comment]
Removing js script in #578 did not seem to work. Emailing Steve Ortiz to see if he has any insight on how to remove the banner.

**Comment by DonNateR** [comment]
Made a new pull request on the source file itself to remove the banner. However, I'm not sure who controls the sources file anymore. Emailed Steve Ortiz to find out.

**Comment by DonNateR** [comment]
@nimbinatus has a possible fix for this. Opening the issue again.

**Comment by nimbinatus** [comment]
PR made in the original repo; @meker12 and I will be trying it in a test environment tomorrow

**Comment by nimbinatus** [comment]
tried a couple hacks; still working on it...

**Comment by nimbinatus** [comment]
figured out why the code kept giving errors; made local fixes and got test server running

**Comment by nimbinatus** [comment]
made a PR on the original repo to fix the issue that was breaking the build and made a PR that will fix the banner

**Comment by nimbinatus** [comment]
Waiting on approval to the last PR, and then for one of the owners to push it up to the CDN.

**Comment by nimbinatus** [comment]
Got all the approvals and the PR has been merged. Now we're tracking down the CDN credentials. Found some contacts with Margaret, and we're en route.

**Comment by nimbinatus** [comment]
I've got access now. Waiting to push the fixes until after business hours (Central Time Zone)

**Comment by nimbinatus** [comment]
Changes posted and live as of this morning (CDT). Please clear your cache if you still see the banner.
